README.md

```diff
@@ -526,6 +528,44 @@ TShark offers a wide variety of services as well as offering a lot of customizat
 
 
 ## References
```

Comment:
I think we could either keep this way or make it a numbered list with the number for each source being inserted at the end of whatever paragraph it was mentioned in.
e.g.
1. https://en.wikipedia.org/wiki/Network_forensics


Network Forensics is a crucial part of digital forensics and its importance continues to grow everyday. Almost all personal computers and servers are going to have built-in networking capabilities, such as wi-fi or ethernet, allowing them to communicate with other devices across the internet. These connections can be used to transfer illegal files, deny normal service, and allow malware to go on and become viral.

Network data can also be difficult to track due to the huge volume and voliticity of conversations between devices.
As a result of this, networking data has to be deliberatly listened to and saved for analysis in order to extract evidence. This is called "sniffing."¹ 

(The 1 is at the end of the second paragraph. I hope this makes sense. If not, or if we don't want to do sources that way, that's ok too!)